Scripts/Spice.py

```diff
@@ -17,14 +19,185 @@ def compute_view_frustrum(instrument):
+    :param correction:  Abberation correction flag
+    :param observingBody: Name of observing body
+    :return: Position of target in a 3D vector
+    :return: light time between the objects as a double
+    """
+
+    return spice.spkpos(mainBody, etTime, referenceFrame, correction, observingBody)
+
+def find_frame_transformation(fromObject: str, toObject:str, etTime: float):
+    """
+    Returns the transformation matrix of how the frame is moving
+
+    Example: find_frame_transformation ( 'IAU_EARTH', 'J2000', ET  )
+
+    ftp://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/pxform.html
+
+    :param fromObject: name of FK of which we are coming from
+    :param toObject: name of FK of which we are going to
+    :param etTime: Epoch of the rotation matrix
+    :return: ET is the amount the position has transformed (magnitude?)
+    :return: The 3x3 transformation matrix M, where (x,y,z)^T *M = (x2,y2,z2)^T
+    """
+
+    return spice.pxform(fromObject, toObject, etTime)
+
+##############
+# DSK Kernel #
+##############
+
+def find_ray_surface_intercept(method: str, target: str, etTime: float, fixedReference:str, correction: str,
+                  observerName: str, directionReference:str, directionVector: list):
+    """
+    Finds where a ray intercepts a surface
+
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/sincpt.html
+
+    :param method: computational method name such as 'ELLIPSOID'
+    :param target: name of the target body
+    :param etTime: et time of the frame
+    :param fixedReference: name of the FK that the target body is fixed to
+    :param correction: Aberration correction
+    :param observerName: the name of the observer taking the frame
+    :param directionReference: name of FK that is the reference frame of which the ray's direction is
+    :param directionVector: Ray direction vector that is from the observer
+    :return: The closest point on the surface that the ray intersects in Cartesion coordinates
+    :return: The intercept epoch, or the distance between the observer and the intercept point
+    :return: Vector from the observer's position at ET to the corrected position of surface intersect point
+    :return: Boolean value which expresses if the ray intersects
+    """
+
+    return spice.sincpt(method, target, etTime, fixedReference, correction, observerName, directionReference,
+                        directionVector)
+
+def find_fov(instrumentID: int, maxReturn: int):
+    """
+    Return information about the instrument
+
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/Tutorials/pdf/individual_docs/25_ik.pdf
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/getfov_c.html
+
+    :param instrumentID: NAIF ID of the instrument (negative six digit number, example: -226801)
+    :param maxReturn: maximum number of vectors that you want to return in 'BOUNDS'
+    :return: shape of the fov, 'POLYGON', 'RECTANGLE', etc...
```

Comment:
`:return` should be a single line description of the return object. Let's put the rest of this information up higher in the docstring.

Reply:
It has 4 return values? If so, write it as a tuple, like in SpiceyPy: https://github.com/AndrewAnnex/SpiceyPy/blob/c71869a311b152ac7c1fabdb6a2bfba58532cb58/spiceypy/spiceypy.py#L11170